src/main/java/org/fundacionjala/at15/katas/langtonant/daniel/Main.java

```diff
@@ -0,0 +1,13 @@
+package org.fundacionjala.at15.katas.daniel.langtonant;
+
+public class Main {
+    
+    public static void main(String[] args) {
+
+        Grid grid = new Grid(6, 6);
+        Ant ant = new Ant(2, 2, grid, "up");
+
+        //grid.printGrid();
+        ant.game(3);
```

Comment:
Looks great. If you can add few more tests (say, bigger grid with ant in a different start position), it would be even better